lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -202,6 +203,19 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         }
 
         /// <summary>
+        /// Returns all the application reports running in the cluster
+        /// </summary>
+        /// <returns></returns>
+        /// TODO: REEF-1824: Implement GetApplicationReports to return the 
```

Comment:
Please reformat all TODO items to `//TODO[REEF-XXXX]: Some text describing them`